test/model/test/common/wiredtiger_util.cpp

```diff
@@ -32,6 +32,7 @@
 #include "wiredtiger.h"
 extern "C" {
 #include "test_util.h"
+#include "../../../../src/log/log_private.h"
```

Comment:
is the path not supposed to be from the include directory specified in the make file.
CMAKE_INSTALL_INCLUDEDIR:PATH=include
so I think it is better the path as "../src/log/log_private.h" instead of relative to this src file.

Reply:
No worries, fixed in [3b1c6c4](https://github.com/wiredtiger/wiredtiger/pull/11107/commits/3b1c6c44c1683ac8c601e383b60682f92b3c8318).